web-components/src/utils/locationStandard.ts

```diff
@@ -79,3 +81,55 @@ export const getJurisdictionIsoCode = ({
     return stateProvince;
   return `${stateProvince} ${_postalCode}`;
 };
+
+/**
+ *
+ */
```

Comment:
Was there supposed to be a comment here?

Reply:
yes, thanks, i updated it!